ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/freemarker/tasks/main.yml

```diff
@@ -30,7 +30,7 @@
     dest: /tmp/
     remote_src: yes
     mode: 0755
-    validate_certs: False
+    validate_certs: True
```

Comment:
Others have all lowercase true - is this deliberate?

Reply:
Not entirely, I've just replaced and matched what was there, I'll switch it to lowercase to align with elsewhere.